rclcpp_components/README.md

```diff
@@ -2,6 +2,22 @@
 
 Package containing tools for dynamically loadable components.
 
+  - [ComponentManager](include/rclcpp_components/component_manager.hpp) Node to manage components. It has the services to load, unload and list current components.
```

Comment:
SInce the other packages link to docs.ros2.org, I think it might make sense to create a docs/FEATURES.md like I did here (https://github.com/ros2/rcpputils/blob/master/docs/FEATURES.md), then add that to the doxygen file. And link to docs.ros2.org here in the README.